src/Test/Proxy/NginxProxy.php

```diff
@@ -15,7 +15,7 @@ class NginxProxy extends Abstractproxy
 {
     protected $binary = 'nginx';
     protected $configFile;
-    protected $port = 8080;
+    protected $port = 8088;
```

Comment:
Hmm. Actually the correct value is `8088`. My tests are not green with `8080` I'm digging on this.